### DISCUSSION TITLE
1.78 update broke this mod.

### DISCUSSION THREAD

**Discussion by MickCasanova** [TitanNano/TS4ControlAnySim#46] [Q&A] [answered]

**Comment by TitanNano** [comment]
Thank you for raising this issue. Unfortunately, I do not currently have the time to investigate what exactly causes the breakage and what is required to fix it. 
The investigation part is sadly the most time-consuming aspect of making the mod compatible again. If anyone can contribute information on what has to be changed or even provide a patch, it would be much easier to get the mod updated.

**Reply by Oops19** [reply]
I'd like to provide a patch but from my point of view the mod is not broken and does not cause exceptions.
Most likely another mod is still partially broken since the update and causes issues for canys.

**Reply by TitanNano** [reply]
@Oops19 so running the Game with only this mod works as before? Could you investigate which mod causes the breakage?

**Reply by Oops19** [reply]
According to another user MCCC is causing the issues and removing MCCC fixes it.
I don't use mods which break with every TS4 update. As I just updated my game I'm quite sure that MCCC will release a fix for the new game version which may or may not fix also the issue with CAnyS.

**Reply by TitanNano** [reply]
@Oops19 CanyS keeps an error / debug log. MCCC has an option to write errors into a log as well. If you or anyone else can provide one or both of these logs, we might be able to figure out what is happening. It's also possible that here is an incompatibility between the two mods.

**Reply by MickCasanova** [reply]
The only issue I had with the mod hanging up on 1.78 was the .json file... Once I deleted it, the mod was fine. People continue to report problems, and this mod has been listed as broken on the sims mod list. I know it works even on 1.81. Now there will be another update tomorrow. Will that break it? Don't know. But I will tell you exactly what I find should it break. For now it continues to work just fine despite my initially reporting an issue that was easily resolved by simply deleting the .json file.

**Reply by MickCasanova** [reply]
I'm pleased to report that so far in 1.82, Control Any Sim appears to work normally. While I haven't played more than in hour or so, one of the first things I did was run CAS though it's paces. I had no issues whatsoever. Of course I will continue to use it, and should an issue develop, I will report it. I certainly can't speak for other's playstyle, but CAS fits mine perfectly. I honestly can't play without it. Can't thank you enough for this gem, TitanNano.

**Comment by TitanNano** [comment, marked as answer]
Mod version 1.2.3 has been released. This version fixes a couple of bugs that could mess up the game. Please report new issues if you encounter problems.
